package.json

```diff
@@ -13,8 +13,8 @@
     "npm": ">=8"
   },
   "scripts": {
-    "build": "webpack --mode=production --color --progress",
-    "start": "webpack-dev-server --config webpack.config.dev.babel.js --hot --progress",
+    "build": "webpack --mode=production --progress",
+    "start": "./node_modules/.bin/webpack-dev-server --config examples/using-module/webpack.config.babel.js --context examples/using-module --open",
```

Comment:
Why do we need to specify `node_modules` here? Doesn't NPM automatically look there?

Reply:
nice catch!